modules/velocloud-sdwan/README.md

```diff
@@ -0,0 +1,92 @@
+| <a name="input_metro_code"></a> [metro\_code](#input\_metro\_code)                                                                                                                                                                                                                                    | Device location metro code. Please refer to [available metros](https://docs.equinix.com/en-us/Content/Interconnection/NE/user-guide/NE-metros.htm)                                                                                                                                                                                                                                                                                               | `string`                                                                      | n/a                                     |   yes    |
+| <a name="input_name"></a> [name](#input\_name)                                                                                                                                                                                                                                                        | Device name                                                                                                                                                                                                                                                                                                                                                                                                                                      | `string`                                                                      | n/a                                     |   yes    |
+| <a name="input_version"></a> [version](#input\_version)                                                                                                                                                                                                                                               | VNF image version. Please refer to [certified VNF versions](https://docs.equinix.com/en-us/Content/Interconnection/NE/user-guide/NE-certified-VNFs.htm) document for the supported versions. If this value is not passed most recent and stable version will be used by invoking [equinix_network_device_software](https://registry.terraform.io/providers/equinix/equinix/latest/docs/data-sources/equinix_network_device_software) data source | `string`                                                                      | n/a                                     |   yes    |
```

Comment:
version_number is the name for the input attribute.

Reply:
fixed